src/client/components/EventCardText/EventCardText.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './EventCardText.styles.css';
+
+export const EventCardText = (props) => {
+  return (
+    <div>
+      <div className="event-card-title">{props.title}</div>
+      <div className="event-card-date-time">{props.datetime}</div>
+      <div className="event-card-mentor">{props.mentor}</div>
+      <div className="event-card-link">{props.link}</div>
```

Comment:
Pass _only_ the link here and add `Link: ` as a part of the component, not as part of the incoming string. Similar to mentor above.